src/contour/contour.yml

```diff
@@ -452,7 +452,15 @@ profiles:
             blur: false
 
         # Specifies a colorscheme to use (alternatively the colors can be inlined).
-        colors: "default"
+        #
+        # This can be either the name to a single colorscheme to always use,
+        # or a map with two keys (dark and light) to determine the color scheme to use for each.
+        # The dark color scheme is used when the system is configured to prefer dark mode and light theme otherwise.
+        #
+        # Default: "default"
+        colors:
+            light: onelight
+            dark: onedark
```

Comment:
It is not clear what `onelight` and `onedark` means

Reply:
just two different color themes that I happen to have locally present. I did not come up with nice default values here. Also, we must provide a default for dark theme and light theme (so not just one) with this PR in the future as well.